src/components/Views/ObservationsView/ObservationsView.js

```diff
@@ -46,14 +46,39 @@ I18n.translations = {
 class ObservationsView extends React.Component<
   Props & StateProps & DispatchProps
 > {
+
+  constructor (props) {
+    super(props)
+    this.state = {
+      observations: []
+    }
+  }
+
+  componentDidMount () {
+    console.log('component mounted')
+    var observable = Observation.list({ limit: 20 })
```

Comment:
This is not really happening yet until we get the outstanding PRs merged in mapeo-core and hyperkv.